app/gui/view/debug_scene/cloud-dashboard/cloud-http/src/lib.rs

```diff
@@ -0,0 +1,200 @@
+    ///
+    /// # Errors
+    ///
+    /// Returns an [`Error`] if:
+    /// - the [`Request`] could not be built,
+    /// - the [`Request`] could not be executed (e.g., due to a network error),
+    /// - the [`Response`] did not have a successful (i.e., 2xx) HTTP status code,
+    ///
+    /// [`Request`]: ::reqwest::Request
+    /// [`Response`]: ::reqwest::Response
+    async fn try_request(&self, route: Route) -> Result<reqwest::Response, Error> {
+        let method = route.method();
+        let relative_path = route.to_string();
+        let mut url = self.base_url.clone();
+        url.push_str(&relative_path);
+
+        let request = self.http.request(method, url);
+        let request = request.bearer_auth(&self.token);
+        let request = request.build()?;
+
+        let response = self.http.execute(request).await?;
+        if !response.status().is_success() {
+            let message = format!("Unexpected response: {response:?}.");
+            Err(message)?
+        }
+
+        Ok(response)
+    }
+}
+
+
+
+// ====================
+// === ApiGatewayId ===
+// ====================
+
+/// Identifier of which Cloud API deployment a request is intended to go to (e.g., a development,
+/// staging, production, or other deployment).
+/// 
+/// This value is the AWS API Gateway ID of the deployment. The API Gateway ID is a unique
+/// identifier generated by AWS when the API is deployed for the first time. It is always available
+/// in the output logs of a successful deployment via our Terraform scripts. It can also be found by
+/// manually navigating to the API Gateway in the AWS web UI.
+#[derive(Clone, Debug, Display)]
+pub struct ApiGatewayId(pub String);
+
+
+
+// =================
+// === AwsRegion ===
+// =================
+
+/// AWS region in which the Cloud API is deployed.
+/// 
+/// This corresponds to the region of the AWS API Gateway identified by the [`ApiGatewayId`] used
+/// when making requests to the Cloud API via the [`Client`]. This value can be found in our
+/// Terraform output logs after a successful deployment of the Cloud API.
+#[derive(Clone, Copy, Debug, Display)]
+#[allow(missing_docs)]
+pub enum AwsRegion {
+    #[display(fmt = "eu-west-1")]
+    EuWest1,
+}
+
+
+
+// ===================
+// === AccessToken ===
+// ===================
+
+/// The JSON Web Token (JWT) used to authenticate requests to our API.
+/// 
+/// Requests without a token are rejected as unauthorized. A token determines which Cloud user is
+/// making a request, what permissions they have, and what resources they can access.
+/// 
+/// The token can be obtained from the browser storage after the user logs in. Do so via the APIs
+/// provided by the [`aws-amplify`] JavaScript library.
+/// 
+/// [`aws-amplify`]: https://docs.amplify.aws/lib/auth/getting-started/q/platform/js/
+#[derive(Clone, Debug, Display)]
+pub struct AccessToken(pub String);
```

Comment:
If we are introducing a strong type for JWT, I'd make its field private and add constructor that does some validation — so we really know that having value `AccessToken` means that it is a token, not some random string.